src/ontology/metadata/icd11.yml

```diff
@@ -0,0 +1,13 @@
+id: ICD11
+label: International Classification of Diseases 11th Revision
+prefix_map:
+  ICD11: http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/ICD11/
+description: >
+  The International Classification of Diseases (ICD) provides a common language that allows health professionals to 
+  share standardized information across the world. The eleventh revision contains around 17 000 unique codes, more than 
+  120 000 codable terms and is now entirely digital.Feb 11, 2022
+  
+  This data source in particular is the ICD11 foundation, not one of its linearizations.
+homepage: https://icd.who.int/
+base_prefix_map:
+  ICD11: http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/ICD11/
```

Comment:
See above

Reply:
This is an artefact of this being branched off of #412, which is still WIP!